packages/components/src/icon-button/index.js

```diff
@@ -41,7 +41,7 @@ class IconButton extends Component {
 		);
 
 		let element = (
-			<Button { ...additionalProps } aria-label={ label } className={ classes }>
+			<Button aria-label={ label } { ...additionalProps } className={ classes }>
```

Comment:
Can't you just use the `label` prop?

Reply:
Not sure what you mean by "use". `Button` doesn't have a `label` prop.